PeaceGeeksSociety/ServicesAdvisor-2.0
# Chore/72 add availability monthly

## Ready State: Yes

This pull request is related to the following JIRA tickets:
* #70 
* #72 
* #73 
* #75 

### Deployment Instructions

* Merge code.
* Run update.php
* Clear caches with `drush cache-clear all`

### Usage/Test Instructions

* Browse to /admin/structure/taxonomy/service_availability, and /admin/structure/taxonomy/service_location_hour_close.
* The term 'Monthly' and 15:00 should appear respectively.
* Browse to  /admin/structure/taxonomy/service_coverage, and /admin/structure/taxonomy/service_intake_criteria.
* The various regions and repeated Youth/ Adolescence should disappear respectively.

### Notes

* Migrated all vocabularies into its own feature (except for Nested Location which will be explained below)
* Exported UUIDs of terms in 'Services Advisor Vocabulary'
* Unable to export translations for terms and vocabularies yet. Arabic translation for #72 will be postponed.
* 'Nested location' and its UUIDs are unable to be migrated successfully
